ui/js/directives/treeherder/clonejobs.js

```diff
@@ -898,11 +906,11 @@ treeherder.directive('thCloneJobs', [
         var generateJobElements = function(resultsetAggregateId, resultset) {
 
             var tableEl = $('#' + resultsetAggregateId);
-
             var waitSpanEl = $(tableEl).prev();
             $(waitSpanEl).css('display', 'none');
-
-            var name, option, platformId, platformKey, row, platformTd, jobTdEl, j;
+            var name, option, platformId, platformKey, j;
+            tableEl.html("");
+            var tableHtml = "";
             for (j=0; j<resultset.platforms.length; j++) {
```

Comment:
If we're going to change so much here, I think we might as well change this to:

```
_.forEach(resultset.platforms, function(platform) {
```

Would save a lot of awkward `resultset.platforms[j]` foo